components/TargetDate.tsx

```diff
@@ -0,0 +1,28 @@
+import moment from 'moment'
+
+type Props = {
+    targetDate?: Date
+}
+
+/**
+ * Renders a datetime as a local date
+ * @param props
+ */
+export default function TargetDate(props: Props) {
+    if (props.targetDate) {
+        return (
+            <time dateTime={moment(props.targetDate).format('YYYY-MM-DD')}>
+                {formatTargetDate(props.targetDate)}
+            </time>
+        )
+    } else {
+        return '-'
+    }
+}
+
+function formatTargetDate(target_date?: Date) {
+    if (!target_date) {

```

Comment:
This check can probably be removed, and we can make `target_date` a required argument for this function. I'd also probably rename it `targetData` so that it matches the variable naming elsewhere in this project.